packages/bot-web-ui/src/components/__tests__/bot-stopped.spec.tsx

```diff
@@ -82,4 +82,26 @@ describe('<BotStopped />', () => {
             expect(global.location.reload).toHaveBeenCalled();
         });
     });
+
+    it('should render onClickClose on press of enter keydown', async () => {
+        const mockEventListener = jest.fn();
+        document.addEventListener('keydown', mockEventListener);
+        render(<BotStopped />, {
+            wrapper,
+        });
+        const close_button = screen.getByTestId('data-close-button');
+        userEvent.type(close_button, '{Enter}');
+        expect(mockEventListener).toHaveBeenCalledWith(expect.objectContaining({ key: 'Enter', code: 'Enter' }));
+    });
+
+    it('should render onClickClose on press of enter keydown', async () => {
```

Comment:
in all tests here we don't need key word`async`, parts such as `await waitFor(() => {` and `act(() => {`
```suggestion
    it('should render onClickClose on press of enter keydown', () => {
```
it works without such things, of course, some of these changes were made before you, but it would be good to clean up since you will do it for your changed lines😌

<img width="1518" alt="Screenshot 2024-03-25 at 17 51 50" src="https://github.com/binary-com/deriv-app/assets/103181650/f8a42273-b6b5-4bc0-a348-89c4d187e99d">

Reply:
Sure @maryia-matskevich-deriv. Will async await and act.